lib/order.rb

```diff
@@ -0,0 +1,87 @@
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  @@tax_rate = 0.075
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+
+    unless fulfillment_status == :pending ||
+           fulfillment_status == :paid ||
+           fulfillment_status == :processing ||
+           fulfillment_status == :shipped ||
+           fulfillment_status == :complete
+      raise ArgumentError, "invalid fulfillment status"
+    end
+  end
+
+  def self.products_hash(products_string)
```

Comment:
Great decision to pull this logic into a helper method, and to make it a class method. Well done!